docs/the-new-architecture/landing-page.md

```diff
@@ -1,32 +1,72 @@
+
+[VisionCamera](https://github.com/mrousavy/react-native-vision-camera), a popular camera library for React Native, leverages this technology to allow real-time frame processing in JavaScript by exposing frame buffers through JSI.
+Typical frame buffers are 10 MB in size, which amounts to roughly 1 GB of data per second, depending on the frame rate. Passing that amount of data through the legacy messaging queue would simply be impossible, whereas with JSI it has almost no overhead at all. JSI can be used to expose other complex instance-based types such as databases, images, audio samples, and more.
+
+Beyond high-volume data processing with JSI, which many libraries have already adopted, JSI adoption in the New Architecture removes this class of serialization work from all native-JavaScript interop.
+
+### Learn more
+
+To achieve this, the New Architecture had to refactor multiple parts of the React Native infrastructure. To learn more about the refactor and other benefits it brings, check out the documentation in the New Architecture working group.
+
+## Benefits of the New Architecture for you
+
+While the New Architecture enables these features and improvements, using the New Architecture in your app or library does not mean guaranteed performance or user experience wins. Data serialization may not have been a bottleneck for your app's performance and certain features may require intentional adoption from your app or third-party dependencies to see benefit.
+
+Overall, enabling the New Architecture in your app or library is opting into the future of React Native as the framework and ecosystem leverage the new capabilities the New Architecture brings.
+
+## So, should I use the New Architecture today?
+
+Today, the New Architecture is considered experimental and we continue to refine backwards compatibility for a better adoption experience.
+
+The team plans to enable the New Architecture by default in an upcoming React Native release by the end of 2024. For most React Native developers, we recommend waiting until that release before enabling the New Architecture.
+
+Our guidance is as follows
+
+- For most production apps, we do _not_ recommend enabling the New Architecture at this time. Waiting for the official release will offer the best experience.
+- If you maintain a React Native library, we recommend enabling it and verifying your use-cases are covered. You can find the [instructions here](https://github.com/reactwg/react-native-new-architecture/blob/lunaleaps-move-docs-over/README.md#documentation).
+
+Due to the major refactor, certain conventions and APIs are no longer supported in the New Architecture. To improve the adoption experience, the team is currently working with library authors and early adopters to catch these use-cases and provide suitable interop between the legacy and New Architecture.
+
```

Comment:
Commenting here but valid to the WG: should we make the New Architecture WG open. Currently users have to go over a Google Fork (linked in the readme) to be included

Reply:
I think that's a good idea, maybe that's something we can remove with this doc landing